frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -113,52 +142,63 @@ function CommonsForm({initialCommons, submitAction, buttonLabel = "Create"}) {
           {errors.milkPrice?.message}
         </Form.Control.Feedback>
       </Form.Group>
+      </Col>
 
-      <Form.Group className="mb-3">
+      <Col xs={6} md={4}>
+      <Form.Group className="mb-3">  
         <Form.Label htmlFor="startingDate">Starting Date</Form.Label>
         <Form.Control
           data-testid={`${testid}-startingDate`}
           id="startingDate"
           type="date"
+          defaultValue={today}
           isInvalid={!!errors.startingDate}
           {...register("startingDate", {
             valueAsDate: true,
             validate: {
-              isPresent: (v) => !isNaN(v) || "Starting date is required",
```

Comment:
I'm not sure what happens here. Is it for testing?

Reply:
Yes. I deleted it cause it was causing me some problems for mutation testing. The same tests where I reset the fields back to empty/invalid doesn't work well for startingDate. Maybe I could try to get the test to look for "mm/dd/yyyy" which signals empty date field but I think no error message is fine for now

Reply:
I guess it still lights up red so it should make sense intuitively. I do think it would be better fixed though